components/shared/infoBox/InfoBox.tsx

```diff
@@ -1,17 +1,39 @@
 import { motion } from "framer-motion"
+import { useRouter } from "next/navigation"
 import React from "react"
 
 import InfoBoxItem from "@/components/shared/infoBox/InfoBoxItem"
-import { getSeriesInfo } from "@/components/shared/infoBox/helper"
-import { WorkFullWorkPageFragment } from "@/lib/graphql/generated/fbi/graphql"
-import { useSelectedManifestationStore } from "@/store/selectedManifestation.store"
+import {
+  ManifestationWorkPageFragment,
+  WorkFullWorkPageFragment,
+} from "@/lib/graphql/generated/fbi/graphql"
+import { resolveUrl } from "@/lib/helpers/helper.routes"
+
+import { Button } from "../button/Button"
 
 type InfoBoxProps = {
   work: WorkFullWorkPageFragment
+  selectedManifestation: ManifestationWorkPageFragment
 }
 
-const InfoBox = ({ work }: InfoBoxProps) => {
-  const { selectedManifestation } = useSelectedManifestationStore()
+const InfoBox = ({ work, selectedManifestation }: InfoBoxProps) => {
+  const router = useRouter()
+
+  const handleClick = (text: string) => {
+    const url = resolveUrl({ routeParams: { search: "search" }, queryParams: { q: text } })
+    router.push(url, {
+      scroll: true,
+    })
+  }
+
+  const ageString = selectedManifestation?.audience?.ages.map(age => age.display).join(", ") || "-"
+  const seriesString =
+    selectedManifestation?.series
+      .map(series => `${series.numberInSeries ? series.numberInSeries + " i " : ""}${series.title}`)
+      .join(", ") || "-"
```

Comment:
I think these functions are so generic that they should be moved globally. Of course only if they are going to be used multiple places. But aren't they?
This is also functionality that we are doing again that we also have done in dpl-react, but I guess that we have already crossed that bridge.